### PR TITLE
Adds eslint with standard configuration

### PR DESCRIPTION
**Pull request by andre-filho**
I added the eslint with standard configuration just as in issue #8

### CODE REVIEW

**Comment by IgorHalfeld** [comment]
Nice! Can you add [eslint-vue-plugin](https://vuejs.github.io/eslint-plugin-vue/developer-guide/#bug-reporting) as well?

**Comment by andre-filho** [comment]
done

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "standard"
```
Comment (IgorHalfeld) [inline .eslintrc.js]: Awesome! Now it's just register :D 
`
"extends": ["plugin:vue/recommended", "standard"]
`

**Review by IgorHalfeld — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (IgorHalfeld) [reply .eslintrc.js]: Can you make this fix? @andre-filho

Comment (andre-filho) [reply .eslintrc.js]: oops my bad

Comment (andre-filho) [reply .eslintrc.js]: it is fixed now

Comment (andre-filho) [reply .eslintrc.js]: btw, I strongly recommend adding package-lock.json to the .gitignore file as it is generated when running `npm install` and make a LOT of conflicts just as it is happening now.

Comment (IgorHalfeld) [reply .eslintrc.js]: Yep! haha.
But the `package.json` is a file like `yarn.lock` (yarn) or `Gemfile.lock` (bundler/ruby), This file exists to actually block the package version, so it needs to be versioned.

Check out [here](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md)

Comment (IgorHalfeld) [reply .eslintrc.js]: Can you fix this?

**Pull request closed without merging**